python/lsst/daf/butler/tests/utils.py

```diff
@@ -87,6 +87,7 @@ def readTable(textTable):
     """
     return AstropyTable.read(textTable,
                              format="ascii",
+                             data_start=2,  # skip the header row and the header row underlines.
```

Comment:
I'm a bit surprised that Astropy doesn't have a parser for the default pretty print table output. Part of me is wondering whether we should add an output format option to all the tabular output subcommands so that users can easily parse the output if they want. That would also let you specify csv format in tests for example.

Reply:
Formatter options are not a bad idea. Another format might be an option to choose between Table and yaml, if there's ever going to be a case where a user may pipe the output of one command to the input of another? How to do naming/data structure with yaml would probably take a little thought though, or maybe you've got good ideas already.

In this case, `AstropyTable.read` normally does the Right Thing, but I ran into a situation where it was getting confused. The table had one column and one row, something like
```
name
----
foo
```
and it was reading the `----` as a value row. I spent a little time trying to figure out why but didn't really get anywhere. It seemed ok to have the unit test function start reading at idx 2 (3rd row, eh) since we're consistent with table output format (until we decide not to be... then we have to come up with a better or more elaborate fix)